packages/blade/src/components/Dropdown/_decisions/decisions.md

```diff
@@ -0,0 +1,702 @@
+# Dropdown & ActionList
+
+Dropdown Menu displays a list of choices on temporary surfaces. They allow users to make a selection from multiple options. They appear when users interact with a button, action, or other control.
+
+<br/>
+
+**Top-View Skeleton**
+
+```jsx
+<Dropdown>
+  <SelectInput label="Item Selection" />
+  <DropdownOverlay>
+    <ActionList>
+      <ActionListItem title="Item Value" value="item-one" />
+      <ActionListItem title="Item Value" value="item-two" />
+    </ActionList>
+  </DropdownOverlay>
+</Dropdown>
+```
+
+<br/>
+
+## Navigation
+
+- [Dropdown](#dropdown)
+  - [SelectInput](#selectinput)
+  - [DropdownOverlay](#dropdownoverlay)
+- [ActionList](#actionlist)
+  - [ActionList Parent Component](#actionlist-parent-component)
+  - [ActionListItem](#actionlistitem)
+  - [ActionListSection](#actionlistsection)
+  - [ActionListHeader](#actionlistheader)
+  - [ActionListFooter](#actionlistfooter)
+- [Why `x` instead of `y`](#why-x-instead-of-y)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [Referrences](#referrences)
+
+---
+
+<br/>
+
+## Dropdown
+
+`Dropdown` will be a wrapper component which will handle the logic of opening and closing of the dropdown. It won't separately render anything in UI apart from the first children. (E.g. `SelectInput` in above example)
+
+### API
+
+Usage
+
+```jsx
+<Dropdown>{/* Dropdown sub-components */}</Dropdown>
+```
+
+Props
+
+_No Props Needed_
+
+### SelectInput
+
+<img src="2022-11-24-16-33-30.png" alt="Select Field Figma Designs" width="500" />
+
+Select is an Input Field that can open a dropdown on click. It can be extended from `BaseInput` component which covers most decisions like validations, focus, etc.
+
+#### Designs
+
+- [Figma - Select](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=13590%3A171090&t=KTWGvEUBEuUdQh9K-0)
+
+#### API
+
+Sample Usage
+
+```jsx
+<SelectInput icon={CalendarIcon} label="Your favorite Design System" helpText="Select only one" />
+```
+
+Props
+
+Mostly similar to other `Input` components extended from `BaseInput`
+
+```ts
+type SelectInputProps = {
+  // ... FormLabelProps like `label`, `labelPosition`
+  // ... FormValidationProps like `validationState`, `helpText`, `errorText`, etc.
+  // ... State Props like `isRequired`, `isDisabled`,
+  icon: IconComponent;
+};
+```
+
+#### A11y
+
+Would be ideal if we can render it as `button` with `aria-expanded` prop to handle state of dropdown open and close.
```

Comment:
I think the `AutoComplete` component would qualify as combobox. In this particular case, we can't really type inside the input. It's just a button that opens and closes the menu.

<img width="805" alt="image" src="https://user-images.githubusercontent.com/30949385/207250900-bcab66d1-e2bd-4727-9ff0-917ac1078fc4.png">

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role

What we have here seems closer to [menu role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitem_role) which seems to recommend button in every example I saw so far.

Reply:
Aha got it. I was confused because the voiceover asks you to "type" words when you're focussing on combobox. 

<img width="1137" alt="image" src="https://user-images.githubusercontent.com/30949385/208657794-4e98be02-4fe5-46b0-98f9-8b037ecaa1de.png">

But w3 link mentions this - 

> However, like an HTML <select>, users can type characters to select matching options.

So I guess they mean typing words to jump on to that option and not exactly typing it inside the input.

I think some other libraries were using menu and not doing this ^^. We can probably do it though. Will update the API decisions

> Would be ideal if we don't handle it ourselves. 

I am really really hoping some library works out lol. So far none are properly working 😢

Reply:
updated it to use combobox. Thanks!